frontend/App.tsx

```diff
@@ -9,41 +9,86 @@ import AuthenticatedPageWrapper from './src/components/AuthenticatedPageWrapper/
 import SignInPage from './src/components/SignIn/SignInPage';
 import { UserProvider, useUser } from './src/components/Providers/UserProvider';
 
+import { MsalProvider } from '@azure/msal-react';
+import { Configuration, PublicClientApplication } from '@azure/msal-browser';
+import { User } from './src/libs/data/src';
+import { Platform } from 'react-native';
+
 export const App = () => {
   const queryClient = new QueryClient();
   const { user } = useUser();
 
   return (
     <QueryClientProvider client={queryClient}>
       <NativeBaseProvider theme={theme}>
-        <NativeRouter>
-          <Routes>
-            {routesConfig.map(({ path, needsAuth, page, footerIcon }) => (
-              <Route
-                key={path}
-                path={path}
-                element={
-                  needsAuth ? (
-                    user ? (
-                      <AuthenticatedPageWrapper footerActiveIcon={footerIcon}>
-                        {page}
-                      </AuthenticatedPageWrapper>
-                    ) : (
-                      <SignInPage />
-                    )
-                  ) : (
-                    page
-                  )
-                }
-              />
-            ))}
-          </Routes>
-        </NativeRouter>
+        <Application user={user} />
       </NativeBaseProvider>
     </QueryClientProvider>
   );
 };
 
+interface AppProps {
+  user: User | null
+}
+
+function Application(props: AppProps): JSX.Element {
+  if (Platform.OS === 'web') {
+    return (
+      <WebApp user={props.user} />
+    );
+  } else {
+    return (
+      <RootApp user={props.user} />
+    );
+  }
+}
+
+function WebApp(props: AppProps): JSX.Element {
+  const configuration: Configuration = {
+    auth: {
+      clientId: process.env.AAD_CLIENT_ID as string,
+      authority: `https://login.microsoftonline.com/${process.env.AAD_TENANT_ID}`,
+      redirectUri: 'http://localhost:19006/sso',
```

Comment:
both props should be moved to some constants file

Reply:
variables was moved to `.env` file